client/src/controllers/index.ts

```diff
@@ -10,6 +10,7 @@ import { SlugController } from './SlugController';
 import { SubmitController } from './SubmitController';
 import { SyncController } from './SyncController';
 import { UpgradeController } from './UpgradeController';
+import { BulkController } from './BulkController';
```

Comment:
not huge, but a reminder that we would like to keep these controllers ordered (as per the comment at the top of this file and below on line 16.

I can fix this up though.